mouse/preview.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org"
+__date__ = "$July 30, 2015 20:20:18 EDT$"
+
+import argparse
+import time
+import sys
+import picamera
+
+
+def main(*argv):
+    parser = argparse.ArgumentParser(
+        description="Takes camera preview. Exit using KeyboardInterrupt."
+        )
+    args =parser.parse_args()
+    with picamera.PiCamera(framerate=90) as camera:
+        try:
+            while True:
+                camera.preview_fullscreen=False
+                camera.preview_window=(100,20,640,480)
+                camera.start_preview()
+                time.sleep(600)
```

Comment:
I actually need it, it flickers on and off if I remove the time sleep function




From: jakirkham <notifications@github.com>
Reply-To: DudLab/mouse_record <reply@reply.github.com>
Date: Friday, August 7, 2015 at 12:45 PM
To: DudLab/mouse_record <mouse_record@noreply.github.com>
Cc: Hwa Bailey <hwab@janelia.hhmi.org>
Subject: Re: [mouse_record] Preview (#8)





In 
mouse/preview.py:
> +import sys
> +import picamera
> +
> +
> +def main(*argv):
> +    parser = argparse.ArgumentParser(
> +        description="Takes camera preview. Exit using KeyboardInterrupt."
> +        )
> +    args =parser.parse_args()
> +    with picamera.PiCamera(framerate=90) as camera:
> +        try:
> +            while True:
> +                camera.preview_fullscreen=False
> +                camera.preview_window=(100,20,640,480)
> +                camera.start_preview()
> +                time.sleep(600)

I don't think we even need it at all because of the while-loop.
�
Reply to this email directly or 
view it on GitHub.